guides/source/getting_started.md

```diff
@@ -583,8 +583,8 @@ the current controller, the `ArticlesController`.
 
 With the form and its associated route defined, you will be able to fill in the
 form and then click the submit button to begin the process of creating a new
-article, so go ahead and do that. When you submit the form, you should see a
-familiar error:
+article, so go ahead and do that (make sure to **reload the page** first to
+apply changes). When you submit the form, you should see a familiar error:
```

Comment:
What changes are you reloading the page in order to apply? It seems like this paragraph is talking about the user typing in text on the browser, not about making code changes - so I'm not sure I understand your edit.

Reply:
This is the current text:

> With the form and its associated route defined, you will be able to fill in the form and then click the submit button to begin the process of creating a new article.

But this is only true if the page is reloaded before clicking the submit bottom, otherwise it will result in a confusing error to beginners (or at least that's how it was for me).

The change we'd apply when we reload IIRC is editing the form to add `url: articles_path`.

Any suggestions on how to make this clearer? Maybe something along these lines (addition in bold):

> Edit the form_with line inside app/views/articles/new.html.erb to look like this:
> `<%= form_with scope: :article, url: articles_path, local: true do |form| %>`
> 
> **Make sure to reload the page to apply the changes.** With the form and its associated route defined, you will be able to fill in the form and then click the submit button to begin the process of creating a new article.

Reply:
PTAL